server/index.js

```diff
@@ -0,0 +1,30 @@
+import express from "express";
+import mongoose from "mongoose";
+import cors from "cors";
+import dotenv from "dotenv";
+import bodyParser from "body-parser";
+import loginRoute from "./routes/logIn.js";
+import signUpRoute from "./routes/signUp.js";
+
+const app = express();
+dotenv.config();
+
+// set them to properly send a reqest
+app.use(bodyParser.json({ limits: "30mb", extended: true }));
+app.use(bodyParser.urlencoded({ limits: "30mb", extended: true }));
+app.use(cors());
+app.use("/login", loginRoute);
+app.use("/signup", signUpRoute);
+
+const DATABASE_URL = process.env.DATABASE_URL;
+const PORT = process.env.PORT || 5000; // for Heroku?
```

Comment:
`process.env.PORT` is for Heroku or any other deploy platform.  I believe wee are going to set PORT number in the platform's environment key setting.
For now, we are going to use post 5000 locally.